Using random target selector (@r) with 1 player causes exception

Using the `@r` target selector when there is only 1 player online may cause this exception:
```
Caused by: java.lang.IndexOutOfBoundsException: Index: 1, Size: 1
        at java.util.ArrayList.rangeCheck(ArrayList.java:653)
        at java.util.ArrayList.get(ArrayList.java:429)
        at net.glowstone.command.CommandTarget.getMatched(CommandTarget.java:242)
```

Glowstone build: 47db4c461f2077620731692777c07e146af4d3f1

Relevant:
https://github.com/GlowstoneMC/Glowstone/blob/47db4c461f2077620731692777c07e146af4d3f1/src/main/java/net/glowstone/command/CommandTarget.java#L239-L246

--- a/src/main/java/net/glowstone/command/CommandTarget.java
+++ b/src/main/java/net/glowstone/command/CommandTarget.java
@@ -234,10 +234,14 @@ public class CommandTarget {
             count = entities.size();
         List<Entity> matched = new ArrayList<>();
         List<Integer> used = new ArrayList<>();
+        Random rand = null;
         for (int i = 0; i < count; i++) {
             if (selector == SelectorType.RANDOM) {
+                if (rand == null) {
+                    rand = new Random();
+                }
                 while (true) {
-                    int random = new Random().nextInt(count + 1);
+                    int random = rand.nextInt(entities.size());
                     if (!used.contains(random)) {
                         matched.add(entities.get(random));
                         used.add(random);

